lib/pages/history_view_page.dart

```diff
@@ -23,6 +24,18 @@ class HistoryViewPageMap extends StatefulWidget {
 class _HistoryViewPageMapState extends State<HistoryViewPageMap> {
   List<Marker> markers = [];
   Position? pos;
+  StreamSubscription<int>? challengeStateStream;
+  
+  @override
+  void initState() {
+    super.initState();
+    challengeStateStream = Global.instance.challengeStateStream.listen((event) {
+      if(event == 1) {
```

Comment:
Would it be possible to use enum instead of an int? for instance an enum that describes the event